tests/test_ranking.py

```diff
@@ -1,26 +1,26 @@
-from nose.tools import eq_
-from tahrir_api.dbapi import TahrirDatabase
-from tahrir_api.model import DeclarativeBase, Assertion
+import unittest
+
 from sqlalchemy import create_engine
 
+from tahrir_api.dbapi import TahrirDatabase
+from tahrir_api.model import Assertion, DeclarativeBase
 
 try:
     from subprocess import check_output as _check_output
 
     def check_output(cmd):
         try:
             return _check_output(cmd)
-        except:
+        except Exception:
             return None
 
-
-except:
+except Exception:
```

Comment:
This block seems to be trying to handle `subprocess.check_output()` not being defined, but it's been there since Python 3.1. In fact, it's now deprecated in favor of `subprocess.run()`.